lib/templates/template.tex

```diff
@@ -1,5 +1,8 @@
 \documentclass[$if(fontsize)$$fontsize$$else$12pt$endif$,$if(lang)$$lang$,$endif$$if(papersize)$$papersize$,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{$documentclass$}
 
+\providecommand{\tightlist}{%
```

Comment:
Further investigation....

It's a command included by default in pandoc's template now:

```
~$ pandoc --print-default-template latex | grep -A 1 tight
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
```